tests/e2e/scripts/cloud-run/deploy.sh

```diff
@@ -121,6 +121,10 @@ function deployBackend() {
 
     sed "s/SERVICE_NAME/${BACKEND_SERVICE_NAME}/g" app_template.yaml > app.yaml
     gcloud app deploy --quiet
+    echo "Sleep 1m for App Engine backend setup"
```

Comment:
Please add a time stamp,  such as
echo $(date)"Sleep 1m for App Engine backend setup"